bg_remove_iterative.py

```
@@ -0,0 +1,414 @@
+    for gy in range(h):
+        for gx in range(w):
+            rgba = layer.get_pixel(gx, gy).get_rgba()
+            r8 = int(rgba[0]*255); g8 = int(rgba[1]*255)
+            b8 = int(rgba[2]*255); a8 = int(rgba[3]*255)
+            if a8 > 5 and g8 > r8 + 8 and g8 > b8 + 4 and g8 > 15:
+                green_pts.append((gx, gy))
+
+    print("SPECKLE_COUNT:" + str(len(green_pts)))
+
+    if green_pts:
+        # Build selection from individual 1×1 rectangles — works on isolated pixels
```

Ruff (0.15.9)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)